master/buildbot/reporters/zulip.py

```diff
@@ -0,0 +1,58 @@
+from twisted.internet import defer
+
+from buildbot import config
+from buildbot.reporters import utils
+from buildbot.reporters.http import HttpStatusPushBase
+from buildbot.util import httpclientservice
+from buildbot.util.logger import Logger
+
+logger = Logger()
```

Comment:
Can we use `log` name like on other reporters?

Reply:
Sorry about that. Rushed into it and forgot about changing the names